ansible/roles/ml-analytics-service/defaults/main.yml

```diff
@@ -4,4 +4,4 @@ BASEPATH: "/opt/sparkjobs" # Git Clone Path **Mandatory
 USER: "data-pipeline"
 WORKDIR: "/opt/sparkjobs"
 SYSTEMDDIR: "/lib/systemd/system"
-ml_analytics_version: "release-4.0.0"
+ml_analytics_version: "release-4.1.0"
```

Comment:
Why do we need this variable here? It's supposed to be given from Jenkins job to clone the repo right?

Reply:
Todo: chanbmge this